src/Windows/Avalonia.Win32/Interop/UnmanagedMethods.cs

```diff
@@ -2411,9 +2420,10 @@ public struct COMDLG_FILTERSPEC
         }
 
         public delegate void MarkFullscreenWindow(IntPtr This, IntPtr hwnd, [MarshalAs(UnmanagedType.Bool)] bool fullscreen);
+        public delegate void SetOverlayIcon(IntPtr This, IntPtr hWnd, IntPtr hIcon, [MarshalAs(UnmanagedType.LPWStr)] string? pszDescription);
         public delegate HRESULT HrInit(IntPtr This);
 
-        public struct ITaskBarList2VTable
+        public struct ITaskBarList3VTable
```

Comment:
Not this PR problem: Looks like something that should be rewritten with MicroCom interop, instead of these vtables.